src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -489,6 +489,29 @@ def _get_extension_suppressions(mod_loaders):
 
         return self.command_table
 
+    @staticmethod
+    def _sort_command_loaders(self, command_loaders):
```

Comment:
`self` should be removed for `@staticmethod`. Otherwise, it fails:

```
ERROR: MainCommandsLoader._sort_command_loaders() missing 1 required positional argument: 'command_loaders'
Traceback (most recent call last):
  File "/mnt/vss/_work/1/s/env/lib/python3.12/site-packages/knack/cli.py", line 233, in invoke
    cmd_result = self.invocation.execute(args)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/vss/_work/1/s/src/azure-cli-core/azure/cli/core/commands/__init__.py", line 564, in execute
    self.commands_loader.load_arguments(command)
  File "/mnt/vss/_work/1/s/src/azure-cli-core/azure/cli/core/__init__.py", line 530, in load_arguments
    command_loaders = self._sort_command_loaders(command_loaders)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: MainCommandsLoader._sort_command_loaders() missing 1 required positional argument: 'command_loaders'
```

```suggestion
    def _sort_command_loaders(command_loaders):
```